core/input/input_map.cpp

```diff
@@ -260,84 +260,444 @@ void InputMap::load_from_project_settings() {
 	}
 }
 
+struct _BuiltinActionDisplayName {
+	const char *name;
+	const char *display_name;
+};
+
+static const _BuiltinActionDisplayName _builtin_action_display_names[] = {
+	/* clang-format off */
+	{ "ui_accept", 									TTRC("Accept") },
+	{ "ui_select", 									TTRC("Select") },
+	{ "ui_cancel", 									TTRC("Cancel") },
+	{ "ui_focus_next", 								TTRC("Focus Next") },
+	{ "ui_focus_prev", 								TTRC("Focus Prev") },
+	{ "ui_left", 									TTRC("Left") },
+	{ "ui_right", 									TTRC("Right") },
+	{ "ui_up", 										TTRC("Up") },
+	{ "ui_down", 									TTRC("Down") },
+	{ "ui_page_up",									TTRC("Page Up") },
+	{ "ui_page_down", 								TTRC("Page Down") },
+	{ "ui_home", 									TTRC("Home") },
+	{ "ui_end", 									TTRC("End") },
+	{ "ui_cut", 									TTRC("Cut") },
+	{ "ui_copy", 									TTRC("Copy") },
+	{ "ui_paste", 									TTRC("Paste") },
+	{ "ui_undo", 									TTRC("Undo") },
+	{ "ui_redo", 									TTRC("Redo") },
+	{ "ui_text_completion_query", 					TTRC("Completion Query") },
+	{ "ui_text_newline", 							TTRC("New Line") },
+	{ "ui_text_newline_blank",						TTRC("New Blank Line") },
+	{ "ui_text_newline_above", 						TTRC("New Line Above") },
+	{ "ui_text_indent", 							TTRC("Indent") },
+	{ "ui_text_dedent", 							TTRC("Dedent") },
+	{ "ui_text_backspace", 							TTRC("Backspace") },
+	{ "ui_text_backspace_word", 					TTRC("Backspace Word") },
+	{ "ui_text_backspace_word.OSX", 				TTRC("Backspace Word") },
+	{ "ui_text_backspace_all_to_left", 				TTRC("Backspace all to Left") },
+	{ "ui_text_backspace_all_to_left.OSX", 			TTRC("Backspace all to Left") },
+	{ "ui_text_delete", 							TTRC("Delete") },
+	{ "ui_text_delete_word", 						TTRC("Delete Word") },
+	{ "ui_text_delete_word.OSX", 					TTRC("Delete Word") },
+	{ "ui_text_delete_all_to_right", 				TTRC("Delete all to Right") },
```

Comment:
Aligning code with tabs is never a good option :)
![Screenshot_20210216_100307](https://user-images.githubusercontent.com/4701338/108040903-2c648300-703e-11eb-9692-5a69c5449bb8.png)

Either use spaces to ensure exact alignment, or remove the alignment.